app/controllers/search_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SearchController < ApplicationController
+
+  def new
+  end
+
+  def index
+    @search_terms = params[:search_terms]
+    @recipes = RecipeApiWrapper.find_recipes(@search_terms).paginate(page: params[:page], per_page: 10)
```

Comment:
You need some sort of check to ensure that `@search_terms` is not null. The same applies to `params[:page]`